src/conv.jl

```diff
@@ -285,8 +285,10 @@ for front_name in (:conv, :∇conv_data, :∇conv_filter,
                         y::AbstractArray{yT,N}, in1::AbstractArray{T1,N},
                         in2::AbstractArray{T2,N}, cdims::ConvDims;
                         kwargs...) where {yT, T1, T2, N}
-            @warn string("Slow fallback implementation invoked for ", $(string(front_name)), "!  ",
+            if yT <: AbstractFloat  # don't print warning for ForwardDiff.Dual
```

Comment:
```suggestion
            if all(T -> T <: G, (yT, T1, T2)) && !(T1 == T2 == yT)  # don't print warning for ForwardDiff.Dual
```

Reply:
My argument against this is mostly that it seems like complexity to define a precise border out in the desert where nobody has been yet. When someone has reason to go there, they'll then need to figure out whether this line was drawn exactly there for what reason... and what's `G`, and why isn't `!(T1 == T2 == yT)` already implied by dispatch... who built that fence... these are sort-of puzzles for future readers. I don't want to presume what "almost passed" means, almost by what metric... this would be much clearer once someone runs into an example.

So my goal was to write the minimal thing, move the line not to cover the case which is actually hit by the tests (Dual) while not removing it from what I think it intends to cover (accidental Float64). 

Anyway, too many words! 

I'd be happy to test `if yT <: Float64` instead, if that sounds better? Super-simple, no puzzle. A mix of Float32 + Float64 will produce this. Pure Float64 won't ever come here. Complex numbers you're on your own. 

Accidental promotion is much less of a problem now that ProjectTo fixes it for you, too.

Reply:
```suggestion
            if yT == Float64  # warn for Float32 + accidental Float64, but don't print warning for ForwardDiff.Dual
```

Reply:
Yes, I see your point. Both the `AbstractFloat` or `Float64` options are fine by me!